src/01-general/work-orders.apib

```diff
@@ -0,0 +1,199 @@
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (WorkOrderData)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + type: `workOrder` (string)
+            + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+
+### workOrders.update [POST /workOrders.update]
+
+Update a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + id: `4afb0a9c-91c6-49ed-a2e5-ce7c1e3a87fb` (string)
+        + Include WorkOrderData
```

Comment:
Following Stijn on this, not the biggest fan of the embedded data structures from the start too 🙈

Reply:
Since we have draft WorkOrders that can be very minimal it does make sense to allow updating most properties on an update, perhaps except the required ones such as customer.

Regarding the embedded data structures, I inlined those on my local branch since it's indeed a hassle when properties (sets) are different on different endpoints.